Makefile

```diff
@@ -161,23 +161,25 @@ test:
 release-update:
 	if [ ! -e RELEASE ]; then echo 0 > RELEASE; fi
 
-version-update: release-update
+.PHONY: ipasetup
+ipasetup:
+	sed -e s/__VERSION__/$(IPA_VERSION)/ ipasetup.py.in \
+		> ipasetup.py
+	$(PYTHON) setup.py egg_info
+	for directory in ipaclient ipalib ipaplatform ipapython ipatests; do \
+	    pushd $${directory} ; \
+	    cp ../ipasetup.py . ; \
+	    $(PYTHON) setup.py egg_info ; \
```

Comment:
Alternatively, if `sys.path` was updated appropriately in each `setup.py`, we would get rid of copying _and_ allow chdir & run `setup.py`.